package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tinymce/eslint-plugin",
-  "version": "1.8.3",
+  "version": "1.9.0",
```

Comment:
Should we consider beehiving this repository?

Reply:
(later, not right now)

Reply:
Yeah, we probably should. I'll log a maintenance task to make sure we don't forget.